legal-api/src/legal_api/config.py

```diff
@@ -332,6 +341,10 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
     TRACTION_PUBLIC_SCHEMA_DID = os.getenv('TRACTION_PUBLIC_SCHEMA_DID', 'TRACTION_PUBLIC_SCHEMA_DID')
     TRACTION_PUBLIC_ISSUER_DID = os.getenv('TRACTION_PUBLIC_ISSUER_DID', 'TRACTION_PUBLIC_ISSUER_DID')
 
+    DOCUMENT_API_URL = os.getenv('DOCUMENT_API_URL', 'http://document-api.com')
```

Comment:
The test config shouldn't ever try to get the actual url. Can you update this to DOCUMENT_API_TEST_URL or just set it to http://document-api.com (I don't think we have a use case where we'd ever want it to actually call anything during testing)

Reply:
hard set to the value that is above